extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -91,7 +99,7 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				}
 			}
 
-			if (shouldPushRecoverableAd) {
+			if (isRecoveryEnabled && isBlocking && adIsRecoverable) {
```

Comment:
Would it be possible to extract method here, or simplify conditions logic in changed lines in this file?